app/screens/Lists/BooksList/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { View, FlatList } from 'react-native';
+
+import BookItem from './components/BookItem';
+import { bookData } from './constants';
+
+class BooksListContainer extends Component {
+  renderList = item => (
+    <BookItem
+      image={item.image_url || undefined}
+      title={item.title}
+      author={item.author}
+    />
+  );
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <FlatList
+          data={bookData}
+          renderItem={({ item }) => this.renderList(item)}
```

Comment:
extract the arrow function like a function of the class to avoid creating a new arrow function instance on each render